### PR TITLE
feat(cloud-bpmn): add variable outline tab

### PR DESCRIPTION
**Pull request by marstamm**
### Proposed Changes

Adds the Variable tab to the desktop modeler. 

image

related to https://github.com/camunda/camunda-modeler/issues/4267

Integration was also tested in the variable-outline PR: https://github.com/bpmn-io/variable-outline/pull/1

 

### Checklist

To ensure you provided everything we need to look at your PR:

* [x] __Brief textual description__ of the changes present
* [x] __Visual demo__ attached
* [x] __Steps to try out__ present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [x] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

**Comment by nikku** [comment]
@jarekdanielak Please have a look into this (review).

File: client/src/app/tabs/cloud-bpmn/BpmnEditor.js [hunk 1]
```diff
@@ -788,6 +790,11 @@ export class BpmnEditor extends CachedComponent {
           engineProfile={ engineProfile }
           onChange={ (engineProfile) => this.engineProfile.set(engineProfile) } />
         }
+
+        <VariableTab
+          id={ this.props.id }
+          layout={ layout }
+          injector={ this.getModeler().get('injector') } />
```
Comment (jarekdanielak) [inline client/src/app/tabs/cloud-bpmn/BpmnEditor.js:797]: Perhaps extract `injector` to a variable like `imported` is in this scope.

**Review by jarekdanielak — APPROVED** [review]
Looks good and works as expected. 👍

File: client/src/app/tabs/cloud-bpmn/BpmnEditor.js (hunk 1 above)
Comment (marstamm) [reply client/src/app/tabs/cloud-bpmn/BpmnEditor.js]: Adjusted in [da0ba48](https://github.com/camunda/camunda-modeler/pull/4459/commits/da0ba489142caa437e38f48d5cb2e9d2e0281784)

**Pull request merged**